www/src/components/IconsTable.scss

```diff
@@ -0,0 +1,84 @@
+.pgn-doc__icons-table {
+  margin-top: 1rem;
+  height: 902px;
+  width: 100%;
+
+  > div {
+    display: inline;
+  }
+
+  .pgn__icon {
+    height: 3rem;
+    width: 3rem;
+    margin: 0 auto;
+  }
+}
+
+.pgn-doc-icons-table__grid {
+  margin: 0 auto;
+}
+
+.pgn-doc__icons-table__cell {
+  @include pgn-box-shadow(1, "centered");
```

Comment:
With the cells flush together, the box-shadow with adjacent cells kind of conflicts. I wonder if the UI might be a bit cleaner without any shadows/borders, except on hover/focus:

<img width="869" alt="image" src="https://user-images.githubusercontent.com/2828721/174301827-8190126e-78cd-4a55-9b2a-3ae69ed11770.png">